### ISSUE TITLE
Add Keyboard shortcut for cycling through tabs

### ISSUE THREAD

**Issue by haacked** [JabbR/JabbR#144]
Thinking `shift+tab` because it's easy using the jquery hotkeys plugin.

**Comment by JesseBuesking** [comment]
Would `shift+tab` cycle forward through the tabs?  When I think `shift+tab`, I think about reverse cycling.  Also, I wouldn't mind doing this one.

**Comment by haacked** [comment]
@JesseBuesking  yeah. But it's hard finding an alternative. It shouldn't be `TAB` because that's reserved for changing element focus in the browser. It shouldn't be `CTRL+TAB` because that's in use for browser tabs.

If you want this, go ahead. :)

**Comment by admsteck** [comment]
What about something like `shift+page up` and `shift+page down` a la Excel?

**Comment by lordscarlet** [comment]
Was this taken care of in 26d0cbff37eefd111dfa6eb91b3a4d1d79a4a31e?

**Comment by dpatti** [comment]
Yes, the current implementation is `tab` to cycle forward, `shift+tab` to cycle backwards. This only works when there is no text in the box, otherwise it would be ambiguous with tab-completion. I don't really think that tabbing between elements is really relevant in JabbR, which is why I did it that way. So far I've found this pretty sufficient for my needs, but if anyone has any feedback or better ideas, I'd love to hear them.
